site/src/modules/resources/SSHButton/SSHButton.tsx

```diff
@@ -92,6 +82,15 @@ export const SSHButton: FC<SSHButtonProps> = ({
 	);
 };
 
+const SSHStep = (props: { helpText: string; codeExample: string }) => (
+	<li style={{ listStylePosition: "inside" }}>
+		<HelpTooltipText style={{ display: "inline" }}>
+			<strong css={styles.codeExampleLabel}>Connect to the agent:</strong>
+		</HelpTooltipText>
+		<CodeExample secret={false} code={props.codeExample} />
+	</li>
+);
```

Comment:
Thanks for reviewing @aslilac 🙏 

Changed to use semantic HTML `<li>` tags _(did it naively to reduce changes, but agree semantic html is better)_

![image](https://github.com/user-attachments/assets/b6a19028-5bba-4aa0-9c7e-95c6fcbdfe67)